samples/reactjs-todo/client/components/journey/webAuthn.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * forgerock-sample-web-react
+ *
+ * webAuthn.js
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+import { FRWebAuthn, WebAuthnStepType } from '@forgerock/javascript-sdk';
+import React, { useEffect, useState } from 'react';
+import Loading from '../utilities/loading';
+
+/**
+ * @function WebAuthn - Footer React component
+ * @returns {Object} - React component object
+ */
+export default function WebAuthn({ step, setSubmissionStep }) {
```

Comment:
I'd advocate for making this more into a complex view, broadening its responsibility. This is what makes this a named stage. Let's move all things WebAuthn into here. Since WebAuthn is deterministic and not dynamic, we don't have to concern ourselves with it being composed with other callbacks or unknown contexts.

Reply:
Overall, i addresed all of the comments and moved the logic into here. There's just one thing i don't yet know how to work out, and this is how to let the parent know it's web-authn flow and hide the key-icon?(above comment)

Reply:
Let me know if the earlier comment clears up how to achieve this goal. Thanks!

Reply:
done